ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
+
+You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can [check the status of various datasets](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) in the data catalogue dashboard. 
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+#### Writing and running code
+
+In Databricks, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+Scripts (e.g. R scripts) can be created and saved in your Workspace and run via the Databricks interface. 
+
+There is detailed guidance on using the Databricks SQL editor, which is similar to SSMS, on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-sql-in-databricks-sql-editor).
+
+One benefit of Databricks is the use of workflows to run your code. Workflows allow you to automate the running of single or multiple scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+---
+
+## What this means for existing code
 
 ---
 
-## Databricks code templates
+### SQL scripts
+
+If you currently use data and scripts stored in SSMS for your statistics publication pipeline, there will be small changes to your production pipelines that vary depending on your current process. 
+
+Your SQL code will need to be migrated over to Databricks to allow it to read from the Databricks SQL Warehouse and will require some minor changes to syntax as SSMS uses T-SQL and Databricks uses Spark SQL. The ADA team are currently developing a tool to allow you to easily convert T-SQL queries to Spark SQL syntax. 
```

Comment:
Does the code need to be migrated to databricks? Or is is more that the code scripts will need translating to a different SQL? This feels a bit confusing depending on where your code is, if it's in an Azure DevOps repo like we recommend for stats then the code itself won't move?

Reply:
I think you have though might be worth combining 'SQL scripts held in repositories and run in a “run all” R script'
and 'SQL code embedded in R scripts' into one called 'SQL code and queries in R and repositories' (or a better name)? 

As I think the steps are actually the same - translate syntax and repoint the connection at a new database but currently they're not written consistently.